src/scenes/AcceleratorRouter/Modules/AddEventModal.tsx

```diff
@@ -0,0 +1,312 @@
+        <Button
+          id='cancel'
+          label={strings.CANCEL}
+          type='passive'
+          onClick={onClose}
+          priority='secondary'
+          key='button-1'
+        />,
+        <Button id='save' onClick={save} label={eventToEdit ? strings.UPDATE : strings.ADD} key='button-2' />,
+      ]}
+      scrolled
+    >
+      <Grid container textAlign={'left'} spacing={2}>
+        <Grid item xs={6} sx={{ marginTop: theme.spacing(2), paddingRight: 1 }}>
+          <DatePicker
+            id='startTime'
+            label={strings.START_DATE}
+            value={record.startTime}
+            onDateChange={(value?: DateTime) => {
+              onChange('startTime', value?.toISO());
+            }}
+            aria-label='date-picker'
+            showTime={true}
+          />
+        </Grid>
+        <Grid item xs={6} sx={{ marginTop: theme.spacing(2), paddingLeft: 1 }}>
+          <DatePicker
+            id='endTime'
+            label={strings.END_DATE}
+            value={record.endTime}
+            onDateChange={(value?: DateTime) => {
+              onChange('endTime', value?.toISO());
+            }}
+            aria-label='date-picker'
+            showTime={true}
+          />
+        </Grid>
+        <Grid item xs={12}>
+          <TextField
+            id='meetingUrl'
+            label={strings.MEETING_URL}
+            type='text'
+            value={record.meetingUrl}
+            onChange={(value: unknown) => onChange('meetingUrl', value)}
+          />
+        </Grid>
+        <Grid item xs={12}>
+          <TextField
+            id='recordingUrl'
+            label={strings.RECORDING_URL}
+            type='text'
+            value={record.recordingUrl}
+            onChange={(value: unknown) => onChange('recordingUrl', value)}
+          />
+        </Grid>
+        <Grid item xs={12}>
+          <TextField
+            id='slidesUrl'
+            label={strings.SLIDES_URL}
+            type='text'
+            value={record.slidesUrl}
+            onChange={(value: unknown) => onChange('slidesUrl', value)}
+          />
+        </Grid>
+        <Grid item xs={12}>
+          <Typography marginTop={2} fontSize='20px' fontWeight={600}>
+            {strings.PROJECTS}
+          </Typography>
+        </Grid>
+        <Grid item xs={12}>
+          {projectsSections.map((ps, index) => {
+            return (
+              <Grid
+                key={`proj-${index}`}
+                container
+                sx={{
+                  borderBottom: `1px solid ${theme.palette.TwClrBrdrTertiary}`,
+                  paddingBottom: 2,
+                }}
+              >
+                <Grid item xs={6} sx={{ marginTop: theme.spacing(2), paddingRight: 1 }}>
+                  <SelectT<CohortModuleWithProject>
+                    id='cohort'
+                    label={strings.COHORT}
+                    placeholder={strings.SELECT}
+                    options={availableCohorts}
```

Comment:
availableCohorts here allow for duplicated selections

Reply:
Fixed